x-pack/plugins/ml/server/models/bucket_span_estimator/bucket_span_estimator.js

```diff
@@ -5,9 +5,13 @@
  */
 
 import cloneDeep from 'lodash/cloneDeep';
+
```

Comment:
Should there be `eslint` commens added here?

Reply:
This is server code so no. I think this was changed by a previous find/replace so we can get it reverted.